contracts/strategies/allocation/AllocationWithVotingBase.sol

```diff
@@ -116,44 +116,53 @@ contract AllocationWithVotingBase is IAllocationStrategy, Ownable, Initializable
     }
 
     modifier onlyPoolAdmin() {
-        if (!registry.isOwnerOrMemberOfIdentity(allo.pools[_poolId].identityId, msg.sender)) {
+        (bytes32 identityId,,,) = allo.pools(poolId);
```

Comment:
This isn't right. Owners are now stored on the Pool.
So we'd have to rely on the OZ roles instead defined in Allo.sol

Reply:
It doesn't look like this is resolved

Reply:
I never pushed (face slap) and you updated 👍🏼